Network-Office/src/main/java/dev/office/networkoffice/chat/service/ChatMessageService.java

```diff
@@ -0,0 +1,79 @@
+package dev.office.networkoffice.chat.service;
+
+import dev.office.networkoffice.chat.dto.request.ChatMessageRequest;
+import dev.office.networkoffice.chat.dto.response.ChatMessageResponse;
+import dev.office.networkoffice.chat.entity.ChatMessage;
+import dev.office.networkoffice.gathering.entity.Gathering;
+import dev.office.networkoffice.gathering.repository.GatheringRepository;
+import dev.office.networkoffice.user.entity.User;
+import dev.office.networkoffice.user.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class ChatMessageService {
+
+    private final GatheringRepository gatheringRepository;
+    private final UserRepository userRepository;
+
+    @Transactional(readOnly = true)
+    public List<ChatMessageResponse> loadChatMessageList(Long roomId) {
+        Gathering gathering = findChatRoomWithMessagesById(roomId);
+//        return gathering.getChatMessageList().stream().map(ChatMessageResponse::from).toList();
+        return gathering.getChatMessageList().stream().map(ChatMessageResponse::anonymousFrom).toList();
```

Comment:
```suggestion
        return gathering.getChatMessageList()
                        .stream()
                        .map(ChatMessageResponse::anonymousFrom)
                        .toList();
```

하나의 라인으로 길게 작성되어서 가독성이 떨어지는 것 같아요. 위와 같이 바꾸거나 이를 분리하는 것은 어떨까요?